util/fqdn.go

```diff
@@ -7,15 +7,15 @@ import (
 	"strings"
 )
 
-func removeQuestionMarks(domain string) string {
+func RemoveQuestionMarks(domain string) string {
 	for strings.HasPrefix(domain, "?.") {
 		domain = domain[2:]
 	}
 	return domain
 }
 
 func IsFQDN(domain string) bool {
-	domain = removeQuestionMarks(domain)
+	domain = RemoveQuestionMarks(domain)
 	if strings.HasPrefix(domain, "*.") {
```

Comment:
Why does one of these get a helper function when the second one doesn't? They have identical logic.

Reply:
They don't - the first runs in a for loop. I can add a helper function if you'd like, but don't think we necessarily need it.

Reply:
```func isPrependedByQuestionMarks(domain string) bool {
	if strings.HasPrefix(domain, "?.") {
		return true
	}
	return false
}```

Reply:
^^ That has the exact same logic there.